src/utils/blockchain.ts

```diff
@@ -22,52 +23,51 @@ export interface AppAuthResult {
   deployerAddress: string;
 }
 
+export const SUPPORTED_CHAINS: Record<number, Chain> = {
+  [mainnet.id]: mainnet,
+  [sepolia.id]: sepolia,
+  [base.id]: base,
+  15107: defineChain({
+    id: 15107,
+    name: "T16Z",
+    nativeCurrency: {
+      name: "ETH",
+      symbol: "ETH",
+      decimals: 18,
+    },
+    rpcUrls: {
+      default: {
+        http: ["https://rpc.t16z.com"],
+      },
+    },
+  }),
+  [anvil.id]: {
+    ...anvil,
+    rpcUrls: {
+      default: {
+        http: ["http://66.220.6.113:8545"],
```

Comment:
it should be `127.0.0.1:8545`

Reply:
So,

```
[anvil.id]: anvil
```

This line is acceptable here.

Reply:
> So,
> 
> ```
> [anvil.id]: anvil
> ```
> 
> This line is acceptable here.

'acceptable here' means?